frontend/src/pages/TermPlanner/main.less

```diff
@@ -1,23 +1,23 @@
 html,
 body {
   height: 100%;
+  --option-header-height: 65px;
```

Comment:
Nice! Like the usage of a css var. Maybe define it inside `App.less` instead? Would be easier to find all css variables when we need to change it in the near future!